muacrypt/mime.py

```diff
@@ -176,10 +196,12 @@ class ACParseResult(object):
     error = attrib_text_or_none()
 
 
-def gen_mail_msg(From, To, _extra=None, Autocrypt=None, Subject="testmail",
-                 Date=None, _dto=False, MessageID=None, payload='Autoresponse\n',
+def gen_mail_msg(From, To, Cc=[], _extra=None, Autocrypt=None,
```

Comment:
usually one avoids using mutable types as default arguments.  in this case ``Cc=[]`` could lead to the function modifying things through e.g. ``Cc.append(...)`` which would modify the next invocation of this function!  It's thus considered bad style.  Rather do ``Cc=None`` and in the function body ``If Cc is None: Cc=[]`` if needed.